src/models/msg.js

```diff
@@ -31,6 +31,7 @@ function Msg(attr) {
 	_.defaults(this, attr, {
 		from: "",
 		id: id++,
+		links: [],
 		previews: [],
```

Comment:
😩
It doesn't seem very logical to me. But I'll try, keep an open mind about it, and see how it looks.